libs/payments/ui/src/lib/nestapp/app.module.ts

```diff
@@ -18,7 +18,12 @@ import {
 import {
   AccountCustomerManager,
   StripeClient,
-  StripeManager,
+  CustomerManager,
+  InvoiceManager,
+  ProductManager,
+  PriceManager,
+  SubscriptionManager,
+  PromotionCodeManager,
```

Comment:
We could refactor this later to have a `billing.module.ts` that contains all its appropriate managers.

Reply:
I'd be interested in going down the modules route for each lib, especially as we have more of these divided responsibility managers.